tests/ArabicdatetimeTest.php

```diff
@@ -8,7 +8,7 @@ class ArabicdatetimeTest extends TestCase
 {
     protected $arabDateTime;
 
-    protected function setUp()
+    public function setUp()
```

Comment:
Using the `public` to be compatible with `sorchestra/testbench` `TestCase class.`